Module/nss/open_barman.nss

```diff
@@ -0,0 +1,17 @@
+//::///////////////////////////////////////////////
```

Comment:
No NWScripts please. Use the existing snippet functionality. https://wiki.starwarsnwn.com/Development/snippets